src/js/view/controlbar.js

```diff
@@ -21,9 +21,47 @@ define([
     }
 
     function menu(name, ariaText) {
-        var createdMenu = new Menu(name, ariaText);
+        return new Menu(name, ariaText);
+    }
 
-        return createdMenu;
+    function reasonInteraction() {
+        return {reason: 'interaction'};
+    }
```

Comment:
`reasonInteraction` is defined twice and breaking the build. Bad merge/rebase. Please fix.